arch/x86_64/mcount-dynamic.c

```diff
@@ -24,10 +24,10 @@ extern void __xray_entry(void);
 extern void __xray_exit(void);
 
 struct xray_instr_map {
-	unsigned long addr;
-	unsigned long entry;
-	unsigned long type;
-	unsigned long count;
+	uint64_t instr_addr;
+	uint64_t func_begin_addr;
+	uint8_t type;
+	uint8_t padding[15];
```

Comment:
Please update this part to be in sync with the LLVM; https://github.com/llvm/llvm-project/blob/d480f968ad8b56d3ee4a6b6df5532d485b0ad01e/compiler-rt/lib/xray/xray_interface_internal.h#L24

Also I'd like to avoid unnecessary changes due to renaming if possible.